fairlearn/grid_search/binary_protected_attribute/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
AFAIK there's no need for __init__.py anymore since they're implicit
https://www.python.org/dev/peps/pep-0420/
That is, unless you want to support super old Python versions. In fact, Python 2.7 will be deprecated in a few months https://pythonclock.org/
I think they added this somewhere in early 3.*
Either way, if you feel like it's worth having just resolve.

Reply:
I had thought they were essential to having things discovered..... let me experiment. It's conceivable that something else which I did allowed everything to hang together

Reply:
It appears that the one in the root of the `test` tree is required, but the others are not